README.md

```diff
@@ -60,6 +64,45 @@ docker cp access.json candigv2_opa_1:/app/permissions_engine/access.json
 
 Restart the OPA container to take effect
 
+## Ingest clinical data
+Note: To upload a DonorWithClinicalData object, you should produce one using the [CanDIG ETL](https://github.com/CanDIG/clinical_ETL_code), then store its result in a JSON file under the key "donors". See single_ingest.json for an example.
+
+### Command line
+Before you can ingest the clinical data, you need to format your data into the json ingest format using the [clinical_ETL](https://github.com/CanDIG/clinical_ETL_data) and put it in the katsu `data` folder, then set the environment variable `CLINICAL_DATA_LOCATION`:
+
+```bash
+export CLINICAL_DATA_LOCATION=path/to/clinical/data/
+```
+
+NOTE: if you want to skip ETL process and use ready-made [synthetic data](https://github.com/CanDIG/katsu/tree/develop/chord_metadata_service/mohpackets/data/small_dataset/synthetic_data), set the path to:
+
+```bash
+export CLINICAL_DATA_LOCATION=https://raw.githubusercontent.com/CanDIG/katsu/develop/chord_metadata_service/mohpackets/data/small_dataset/synthetic_data/
+```
+
+Reload the environment variables:
+
+```bash
+source env.sh
+```
+
+Run katsu_ingest.py script. This will represent several options:
+
+1. Run check: check if ingest is possible
+2. Ingest data: import the data into katsu
+3. Clean data: reset the database to the clean state. Use this if you want to start over. WARNING: be careful in production environment, as it irreversible.
+4. Ingest DonorWithClinicalData: Ingest a DonorWithClinicalData object into Katsu - CLINICAL_DATA_LOCATION should be a single JSON file
+
+You can also run the script with the `-choice` to skip the menu and go straight to the choice you want.
+
+```bash
+python katsu_ingest.py -choice 2
+```
+
+### API
+The clinical ingest API runs at /ingest/clinical_donors. Simply send a request with an authorized bearer token and a JSON body with your DonorWithClinicalData object. See the swagger UI/schema for the response format.
+
+
 ## Ingest genomic files
 
 ### Genomic file preparation
```

Comment:
would be good to have a note at the top of this section that clinical data needs to be ingested before genomic data